_studio/shared/umc/codec/jpeg_dec/src/umc_mjpeg_mfx_decode_base.cpp

```diff
@@ -324,15 +324,15 @@ Status MJPEGVideoDecoderBaseMFX::_GetFrameInfo(const uint8_t* pBitStream, size_t
     int32_t   precision;
     JSS      sampling;
     JCOLOR   color;
-    CMemBuffInput in;
+    CMemBuffInput* in = new CMemBuffInput();
```

Comment:
Is that new corresponding to delete in the prev. patch? If yes, prev. patch looks wrong to me!